python/ray/util/iter.py

```diff
@@ -289,6 +291,9 @@ def repartition(self, num_partitions: int) -> "ParallelIterator[T]":
         Args:
             num_partitions (int): The number of shards to use for the new
                 ParallelIterator
+            batch_time (int): Batches items for batch_time milliseconds
+                on each shard before retrieving it.
+                Increasing batch_time reduces latency but improves throughput.
```

Comment:
I think you mean increases latency? 

Decreasing latency and improving throughput sounds great :)